R/learner_xgboost_surv_xgboost.R

```diff
@@ -12,7 +12,9 @@
 #' - `nrounds` is initialized to 1.
 #' - `nthread` is initialized to 1 to avoid conflicts with parallelization via \CRANpkg{future}.
 #' - `verbose` is initialized to 0.
-#' - `objective` is initialized to `survival:cox` for survival analysis.
+#' @section Custom mlr3 parameters:
```

Comment:
why is this a custom parameter and not an initial value?

Reply:
the `type` argument is an initialization parameter but mlr3-specific, though it affects one of the parameters (`objective`). I think that it is better to delete the `objective` in this case and specify only the `type` to make the split consistent (see issue in the comment above). What do you think?